R/infrastructure.R

```diff
@@ -399,40 +399,75 @@ use_data <- function(..., pkg = ".", internal = FALSE, overwrite = FALSE,
   pkg <- as.package(pkg)
 
   to_save <- dots(...)
```

Comment:
@hadley: Do you mind if I import `lazyeval` and implement a corresponding `use_data_` function? The above seems to be the only remaining instance of `dots()`.